drivers/i2c/i2c_ll_stm32_v2.c

```diff
@@ -136,6 +151,7 @@ int stm32_i2c_msg_write(struct device *dev, struct i2c_msg *msg,
 	data->current.is_write = 1;
 	data->current.is_nack = 0;
 	data->current.is_err = 0;
+	data->current.msg = msg;
```

Comment:
I might have missed an element, but I don't see why this was required..

Reply:
in line 97 the isr needs to know if I2C_MSG_STOP flag was set. data->current.msg was available already, but unused so far.

Reply:
ok, thanks.